src/main/scala/chisel3/tester/package.scala

```diff
@@ -39,11 +40,19 @@ package object tester {
         require(x.binaryPoint == value.binaryPoint, "binary point mismatch")
         pokeBits(x, value.litValue)
       }
-      case (x: Bundle, value: Bundle) => {
+      case (xx: Bundle, value: Bundle) => {
         // TODO: chisel needs to expose typeEquivalent
-        require(x.elements.keys == value.elements.keys)  // TODO: this discards type data =(
-        (x.elements zip value.elements) foreach { case ((_, x), (_, value)) =>
-          x.poke(value)
+        require(xx.elements.keys == value.elements.keys)  // TODO: this discards type data =(
+        (xx.elements zip value.elements) foreach { case ((fieldName, xxx), (_, value)) =>
```

Comment:
I'm pretty sure chaining more `x`s into a variable name does not make it better

Reply:
It's better than all of them being the same name. This was particularly complicated when I did some work for Adam and Jack on cleaning up style in some older firrtl sections. It was difficult to figure out which var was in play in many cases.